src/pages/MenuPage.js

```diff
@@ -24,7 +64,24 @@ export class MenuPage extends React.Component {
   }
 
   componentWillMount() {
-    const { database } = this.props;
+    const { database, settings } = this.props;
+
+    this.sections = SECTIONS;
+    // check is there are any modules enabled in this store customData
+    const usableModules = this.getUsedModules(settings.get(THIS_STORE_CUSTOM_DATA));
+    const usesModules = usableModules.length > 0;
+    this.sections.modules.buttons = usableModules;
```

Comment:
Would be good to have these initialized to null in the constructor. I think it makes it a lot clearer rather than random class scoped variables popping up

Reply:
done: https://github.com/openmsupply/mobile/commit/cdc1d9034ff584383b6407ad421f324e2a2bf9f2